fcrepo-kernel-modeshape/src/main/java/org/fcrepo/kernel/modeshape/FedoraResourceImpl.java

```diff
@@ -1022,4 +1025,21 @@ protected Property getProperty(final String relPath) {
         }
     }
 
+    /**
+     * A method that takes a Triple and returns a Triple that is the correct representation of
+     * that triple for the given resource.  The current implementation of this method is used by
+     * {@link PropertiesRdfContext} to replace the reported {@link org.fcrepo.kernel.api.RdfLexicon#LAST_MODIFIED_DATE}
+     * with the one produced by {@link #getLastModifiedDate}.
+     */
+    public static Triple fixTriples(final FedoraResource r, final Converter<Node, Resource> translator,
```

Comment:
In addition to @acoburn's comments, I would suggest changing the name of this method to be more descriptive of what is happening... along the lines of `fixDatesIfNecessary()`... or the like.